### PR TITLE
Create 2023 election documents

### LINKED ISSUE #1483: Create the directory on this repository for the election
Use previous years as a template

### PR DESCRIPTION
**Pull request by dyladan**
Fixes #1483

### CODE REVIEW

File: elections/2023/governance-committee-election.md
Comment (dyladan) [inline elections/2023/governance-committee-election.md:99]: @open-telemetry/otel-elections we need to decide on these dates

**Comment by dyladan** [comment]
Link check fails because it points at the candidate list which is created by this PR

File: elections/2023/governance-committee-election.md
Comment (mtwo) [inline elections/2023/governance-committee-election.md:34]:
```suggestion
* [Morgan McLean](https://github.com/mtwo), Splunk, until October 2024
```

**Review by mtwo — APPROVED** [review]

File: elections/2023/governance-committee-election.md
Comment (yurishkuro) [inline elections/2023/governance-committee-election.md:84]: Can anyone login & vote with GitHub or only a predetermined list (of eligible members)?

Comment (jpkrohling) [reply elections/2023/governance-committee-election.md:84]: The election committee will upload a list of github usernames that are eligible for voting.

File: elections/2023/governance-committee-election.md
Comment (jpkrohling) [inline elections/2023/governance-committee-election.md:105]:
```suggestion
| 23 October 2023 00:00 UTC | Voting period begins                                                                                                                       |
```

File: elections/2023/governance-committee-election.md
Comment (jpkrohling) [inline elections/2023/governance-committee-election.md:104]: I recommend this being a couple of days before the voting period. While it was uneventful to have it the day before in previous elections, I think it's safer to allow for a buffer...

**Comment by dyladan** [comment]
Updated dates. Please take a look to make sure I haven't made any mistakes.

File: elections/2023/governance-committee-election.md
Comment (jpkrohling) [inline elections/2023/governance-committee-election.md:20]: When I read that the deadline is 13 October, my mind registers that I can still submit my application on that date. So, the deadline would be 13 October 2023 23:59 UTC instead?

```suggestion
* If you've been nominated or are willing to nominate yourself: check the [charter document](../../governance-charter.md) and confirm you are ready for the commitment. Make sure to provide all necessary information before 13 October 2023 23:59 UTC
```

File: elections/2023/governance-committee-election.md
Comment (jpkrohling) [inline elections/2023/governance-committee-election.md:22]: Can you double-check the dates, verifying it matches Helios?

image

File: elections/2023/governance-committee-election.md
Comment (jpkrohling) [inline elections/2023/governance-committee-election.md:41]: It's time to remove this paragraph. It's from a time when we were changing the size of the GC.

File: elections/2023/governance-committee-election.md
Comment (jpkrohling) [inline elections/2023/governance-committee-election.md:92]:
```suggestion
Voting will close on 25 October 2023 at 23:59 UTC. Nominees will be stack ranked. If a nominee becomes ineligible (for instance, if more than three topmost nominees work for the same company), the election committee will skip those nominees and pick the nominee with the next-highest score. The exact scores for each candidate will be public.
```

**Review by jpkrohling — APPROVED** [review]

**Comment by jpkrohling** [comment]
I would still like to have @trask's approval before merging this, given he's at the election committee.

**Review by trask — APPROVED** [review]

**Pull request merged**
